pyproject.toml

```diff
@@ -63,10 +62,12 @@ ignore = [
     "C408",
     "C901",
     "B904",
+    "UP033", # Ignore the lru_cache for now
+    "UP017", # Ignore datetime.UTC for now
 ]
 unfixable = [
-    # Don't touch unused imports
-    "F401",
+    "UP033", # Don't fix the lru_cache yet
+    "UP017", # Don't fix datetime.UTC yet
```

Comment:
To reviewers: this is to avoid hard-breaking people that are still using python 3.8 and 3.9 locally so they have the time to upgrade